src/main/java/io/sealights/plugins/sealightsjenkins/BeginAnalysis.java

```diff
@@ -689,13 +689,15 @@ private void setParamByKey(Map<String, String> paramsMap, String key, SeaLightsP
             slInfo.setTestListenerJvmParams(paramsMap);
     }
 
-    private String resolveBuildSessionId(Logger logger, SeaLightsPluginInfo slInfo, Properties additionalProps) {
+    private String resolveBuildSessionId(Logger logger, SeaLightsPluginInfo slInfo, Properties additionalProps,
+     EnvVars envVars) {
 
         resolveCreateBuildSessionIdProperty(slInfo, additionalProps);
 
         String buildSessionIdFile = (String) additionalProps.get("buildsessionidfile");
 
         ArgumentFileResolver argumentFileResolver = new ArgumentFileResolver();
+        buildSessionId = envVars.expand(buildSessionId);
```

Comment:
What happens with BSID File? Do you resolve that as well?

Reply:
Fixed